tests/cpu_bugs/xen_grub_setup.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+
+use warnings;
+use strict;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use bootloader_setup 'replace_grub_cmdline_settings';
+
+sub run {
+    my $self  = shift;
+
+    replace_grub_cmdline_settings('unit=0', 'unit=1', update_grub => 1, search => '^GRUB_SERIAL_COMMAND=');
```

Comment:
if setting GRUB_SERIAL_COMMAND, does still need to set GRUB_CMDLINE_XEN_DEFAULT? , just like this example "
GRUB_CMDLINE_XEN_DEFAULT="console=com1 com1=115200"
"

Reply:
Based on my verification in [10.67.134.217/tests/11364](url), we no need set GRUB_CMDLINE_XEN_DEFAULT